src/encoded/tests/test_access_key.py

```diff
@@ -139,4 +139,4 @@ def test_access_key_uses_edw_hash(app, access_key):
     root = app.registry[COLLECTIONS]
     obj = root.by_item_type['access_key'][access_key['access_key_id']]
     pwhash = obj.properties['secret_access_key_hash']
-    assert EDWHash.encrypt(access_key['secret_access_key']) == pwhash
+    assert EDWHash.hash(access_key['secret_access_key']) == pwhash
```

Comment:
EDWHash should definitely be removed from snovault. Ideally it would be removed from encoded as well but that may require a staged approach.

1. Configure the crypt context to use a recommended hash (https://passlib.readthedocs.io/en/stable/narr/quickstart.html) by default and use the new hash for newly created access keys.

2. Optionally migrate existing hashes to the new hash as they are used (probably requires logging the old and new hashes along with the access key as we don't update the db on read.)

3. Once enough hashes are migrated over removed the old hash and perhaps contact any people who have not used their access key in the migration period that they must reset their access key (generating a new password) before they use it next time.